### ISSUE TITLE
'Stabl' object has no attribute 'pairwise'

### ISSUE THREAD

**Issue by tri-dataxight** [gregbellan/Stabl#18]
Hi team. I'm having an issue while following a script provided in the repo.

## How to reproduce
* Follow the tutorial at: https://github.com/gregbellan/Stabl/blob/v1.0.0/Notebook%20examples/run_cv_COVID.py
* Stable version: `Stabl @ git+https://github.com/gregbellan/Stabl.git@92cef79e4c3f92d9b717b1c9eaaffbcc321142be` (installing with `pip install git+https://github.com/gregbellan/Stabl.git@v1.0.0-lw`)

## Errors/Screenshots:
* An error will occur when executing this function:
```python
multi_omic_stabl_cv(
        data_dict=X_train,
        y=y_train,
        outer_splitter=outter_cv,
        estimators=estimators,
        task_type=task_type,
        save_path="./Benchmarks results/Results COVID-19",
        outer_groups=ids,
        early_fusion=True,
        late_fusion=True,
        n_iter_lf=10, # Micro test
        models=models
)
```
<img width="1325" alt="Screenshot 2024-11-18 at 14 55 23" src="https://github.com/user-attachments/assets/236a787c-f2d2-4edd-9191-17c5c3ec2d7f">

**Comment by xavdurand** [comment]
Hello @tri-dataxight ,

Thank you for your issue. 
This error was solved in the new version of the library as described in the issue #7: 

 > I can confirm that this should be fixed in version 1.0.1. I also had the same issue because I followed the `README` suggestion to install `pip install git+https://github.com/gregbellan/Stabl.git@v1.0.0-lw`, which points to the previous version. Probably worth updating?
> _Originally posted by @FrenkT in https://github.com/gregbellan/Stabl/issues/7#issuecomment-2425132291_

As it was solved, I close the issue.

**Comment by tri-dataxight** [comment]
> Hello @tri-dataxight ,
> 
> Thank you for your issue. This error was solved in the new version of the library as described in the issue #7:
> 
> > I can confirm that this should be fixed in version 1.0.1. I also had the same issue because I followed the `README` suggestion to install `pip install git+https://github.com/gregbellan/Stabl.git@v1.0.0-lw`, which points to the previous version. Probably worth updating?
> > _Originally posted by @FrenkT in [#7 (comment)](https://github.com/gregbellan/Stabl/issues/7#issuecomment-2425132291)_
> 
> As it was solved, I close the issue.

I see. Thanks. You definitely should update the README of the `main` branch.
